doc/rst/source/cookbook/introduction.rst

```diff
@@ -80,10 +80,11 @@ cookbook which explains the purpose of the package and its many
 features, and provides numerous examples to help new users quickly
 become familiar with the operation and philosophy of the system. The
 cookbook contains the shell scripts that were used for each example;
-PostScript files of each illustration are also provided. All programs
-have individual manual pages which can be installed as part of the
-on-line documentation under the UNIX **man** utility or as web
-pages. In addition, the programs offer friendly help messages which make
+PostScript files of each illustration are also provided. The online 
```

Comment:
I dont think we distribute PostScript files, so perhaps just remove that sentence - it seems dated.

Reply:
Removed in 2d7a7fa